modules/release-notes/partials/docs-server-7.2.6-release-note.adoc

```diff
@@ -34,7 +34,7 @@ For details on new features and enhancements, see the  xref:introduction:whats-n
 | Issue resolved.
 
 | https://jira.issues.couchbase.com/browse/MB-63286[MB-63286]
-| An issue occurred caused by a race condition in the index recovery code path, which may result in item count mismatch and wrong query results. This  issue may occur during an Indexer restart.  However, as part of the file-based rebalance process introduced in `7.6.0`, a recovery of the index is performed after the index is moved, which increases the likelihood that this race condition might be reached.
+| An issue occurred caused by a race condition in the index recovery code path, which may result in item count mismatches and incorrect query results. This  issue may occur during an Indexer restart.
```

Comment:
I think we need to tweak the sentence (we have some past tense, some present tense).

An issue occurred, caused by a race condition in the index recovery code path, which sometimes resulted in item count mismatches and incorrect query results. The issue sometimes occurred during an Indexer restart.

Reply:
Done.